libs/libvtrutil/src/vtr_geometry.h

```diff
@@ -3,6 +3,8 @@
 #include "vtr_range.h"
 #include "vtr_assert.h"
 
+#include <stdio.h> // vtr_geometry.tpp uses printf()
```

Comment:
Should probably be `cstdio`?

Reply:
Looks like it is common in this directory, so let's use `<cstdio>`. Updated.
(usually, I recommend `stdio.h` for cleanly stating the origins, but seem that this part of the project is not doing that)

Reply:
In C++, `cstdio` should always be preferred, at least by the standard: https://stackoverflow.com/questions/7596406/stdio-h-not-standard-in-c

Reply:
Yes, but it is important to know the rules to know when it is ok and better to break them :D